app-devices/yubioath-flutter/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=yubioath-flutter
+PKGSEC=admin
+PKGDEP="ccid pcsclite yubikey-manager gtk-3 libappindicator libnotify"
+BUILDDEP="swig fvm uv llvm desktop-file-utils"
+PKGDES="Yubico Authenticator for Desktop"
+PKGREP="yubioath-desktop"
+USECLANG=1
+# FIXME: Need to be tested on these architectures
```

Comment:
This is not a FIXME. What we need to let others know here is the reason of excluding other architectures.